chain-alerter/src/main.rs

```diff
@@ -42,12 +48,23 @@ const REQUEST_THROTTLE: Duration = Duration::from_secs(2);
 const MAX_SLACK_API_RETRIES: usize = 30;
 
 /// The Autonomys Slack Team ID. This is not a secret.
-/// TODO: if we ever operate the bot in multiple workspaces, make this a configurable env/CLI parameter
+/// TODO: if we ever operate the bot in multiple workspaces, make this a configurable env/CLI
+/// parameter
 const AUTONOMYS_TEAM_ID: &str = "T03LJ85UR5G";
 
+/// One Subspace Credit, copied from subspace-runtime-primitives.
+const AI3: u128 = 10_u128.pow(18);
+
+/// The maximum balance change to alert on.
+const MAX_BALANCE_CHANGE: u128 = 1_000_000 * AI3;
```

Comment:
@jfrank-summit how large do you want the transfer threshold to be?

(This doesn't cover XDMs yet, but it's useful as a first step.)

Reply:
1,000,000 is fine to start with. This naming seems backwards to me, wouldn't we want to alert on a `MIN_BALANCE_CHANGE`?